.github/ISSUE_TEMPLATE/4_documentation_change.md

```diff
@@ -0,0 +1,21 @@
+---
+name: Documentation change request
+about: Report documentation issue or request documentation changes
+title: ''
+labels: documentation
+assignees: ''
+
+---
+
+**Summary**
+Include a short summary of the issue or request. 
+See the sections below
```

Comment:
This seems redundant. I think that with a form like this, our users know to fill out the info below. I would remove the "See the sections below for factors important for a documentation issue."

Reply:
Done.